specification/billing/resource-manager/Microsoft.Billing/preview/2018-05-31-preview/examples/ProjectsListByBillingAccountId.json

```diff
@@ -0,0 +1,33 @@
+{
+  "parameters": {
+    "api-version": "2018-05-31-preview",
+    "billingAccountId": "{billingAccountId}",
+    "$filter": "permission eq 'canCreateSubscription'"
```

Comment:
Are we sure we're proceeding with this?

Reply:
Good catch. I have corrected this as well.